neonvm/pkg/ipam-demo.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+	"time"
+
+	"go.uber.org/zap/zapcore"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
+	"github.com/neondatabase/autoscaling/neonvm/pkg/ipam"
+)
+
+var (
+	kconfig = flag.String("kube-config", "~/.kube/config", "Path to kuberenetes config. Only required if out-of-cluster.")
```

Comment:
Is this needed? If so, can it just use the `$KUBECONFIG` environment variable?

Reply:
[clientcmd.BuildConfigFromFlags](https://pkg.go.dev/k8s.io/client-go/tools/clientcmd#BuildConfigFromFlags) function used in IPAM code doesn't support it. I will rewrite that part in next PRs

Reply:
It seems like this should be a simple change to `ipam-demo.go`, no? I'm not sure what the blockage is. Is there a reason we can't just replace this with checking the environment variable, instead of a flag

https://github.com/neondatabase/autoscaling/blob/d6aec919963dbab4715ef1d44bbea46d1aa30124/neonvm/pkg/ipam-demo.go#L50-L55

Reply:
added to follow ups list